src/repositories/docsetsRepository.ts

```diff
@@ -52,9 +52,11 @@ export class DocsetsRepository extends BaseRepository {
     ];
   }
 
-  async getProjectByRepoName(repoName: string): Promise<any> {
+  async getProjectByRepoName(repoName: string, directory?: string): Promise<any> {
```

Comment:
NIT: This is opening up a can of worms, and I know that this isn't your change specifically, but we should provide a different return type than `Promise<any>`. Specifically, let's replace the `any` with a more concrete type.

The reason why this is opening up a can of worms is because of how we type the collection, which is assigned a type annotation in the super class... That might be a bit of work, so don't feel like you need to get this changed for this PR.

Reply:
Great callout!

Reply:
Just checked, and I believe that is correct!